docs/quickstart.md

```diff
@@ -131,6 +133,18 @@ ark restore describe <RESTORE_NAME>
 
 For more information, see [the debugging information][18].
 
+### View logs
+
+To view logs from local storage, you need to modify your configuration.
+
+1. In `examples/minio/00-minio-deployment.yaml`, change the value of `spec.type` in the `Service` YAML from `ClusterIP` to `NodePort`.
+
+1. Run `kubectl -n heptio-ark get svc/minio -o jsonpath='{.spec.ports[0].nodePort}'` to get the value of the NodePort address.
+
+1. In `examples/minio/05-ark-backupstoragelocation.yaml`, change the value of `spec.config.s3Url` to the value of the NodePort address returned in the previous step.
```

Comment:
The NodePort only contains a port, such as 35464. The value of the s3Url needs to be a fully valid URL, including http:// or https://. They can use the DNS name or IP address of any of the nodes in the cluster.